DEV-9/EntryPoint.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ReplacementOfCharacterGroups
+{
+  // Main class method reads two sequences of characters from the file
+  // and calls a method to randomly replace groups of characters.
+  public class EntryPoint
+  {
+    // Argument of method Main contains the path to the file.
+    static void Main(string[] args)
+    {
+      const int seqCount = 2;
+      try
+      {
+        List<string> seqList = new TextFileReader().ReadLines(args[0]);
+
+        if (seqList.Count < seqCount)
+        {
```

Comment:
1. Not the best choice for exception type
2. What about seqList.Count > seqCount?